extensions/data-workspace/src/dialogs/projectDashboard.ts

```diff
@@ -0,0 +1,124 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as constants from '../common/constants';
+import { IProjectAction, IProjectActionGroup, IProjectProvider, WorkspaceTreeItem } from 'dataworkspace';
+import { IWorkspaceService } from '../common/interfaces';
+
+export class ProjectDashboard {
+
+	private dashboard: azdata.window.ModelViewDashboard | undefined;
+	private modelView: azdata.ModelView | undefined;
+	private projectProvider: IProjectProvider | undefined;
+	private overviewTab: azdata.DashboardTab | undefined;
+
+	constructor(private workspaceService: IWorkspaceService, private treeItem: WorkspaceTreeItem) {
```

Comment:
private vars typically start with `_`

Reply:
They are?  I've seen without typically, and can't find mention of that in typescript styleguides.  Over in the C# world, that's a C++ism that they're slowly shedding.

Reply:
For us they do. I can't really speak to the larger community. Where do you "typically" see them without an underscore?

Part of the problem is that we're not super consistent - but I'd definitely say that for newer code we write that we almost always default to underscore.

Reply:
I have definitely seen an _ before private vars in our code. Modified the code for next iteration.